src/sbt-test/sbt-scalafix/basic/build.sbt

```diff
@@ -5,6 +5,10 @@ inThisBuild(
       "org.scalameta" %% "testkit" % "4.0.0-M6" % Test,
       "org.scalacheck" %% "scalacheck" % "1.13.5" % Test,
       "org.scalatest" %% "scalatest" % "3.0.5" % Test
+    ),
+    scalafixDependencies := List(
+      // Custom rule published to Maven Central https://github.com/olafurpg/example-scalafix-rule
+      "com.geirsson" % "example-scalafix-rule_2.12" % "1.1.1"
```

Comment:
Should be `"com.geirsson" %% "example-scalafix-rule" % "1.1.1"`

Reply:
That won't work for the scripted tests on 0.13

Reply:
Fair point.